apps/admin-gui/src/app/vos/pages/member-detail-page/member-detail-page.component.html

```diff
@@ -9,6 +9,13 @@ <h1 class="page-title mb-1">
     <span class="text-muted">
       &nbsp;#{{member.id}}
     </span>
+    <br>
+    <span class="text-muted mt-1 entity-info">
+      {{'MEMBER_DETAIL.USER_ID' | translate}}:
+      <span class="member-text">
+        {{member.userId}}
```

Comment:
I would like the id of user be clickable and lead to admin section - user overview(lead into user detail)